roles/host-provision/tasks/rhel-config-tasks.yml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: config-tasks | Validate internet connectivity via control node
+  uri:
+    url: https://www.google.com
+    status_code: 200
+  tags: host-provision
+
+- name: config-tasks | Get current registration status
```

Comment:
It may be helpful to have an explanatory comment that we explicitly check and evaluate subscription status to avoid prompting for RedHat credentials unless necessary.

Reply:
Done